### PR TITLE
Support for unblocked user tags

### LINKED ISSUE #9: When tagging the parameter "notblocked" is always set to "no", regardless if the sock is blocked or not
See https://en.wikipedia.org/w/index.php?title=User:DJ_S_UNIVERSE_Q_(1)&action=edit&oldid=991336133 where the tag was placed. The sockpuppet was not locally blocked, but was globally locked. If the sock is not blocked, "notblocked" should be set to yes. If the sock is blocked, it should be set to no.

### PR DESCRIPTION
**Pull request by Dreamy-Jazz**
Add support for users who are not blocked, but tagged
Fixes #9

### CODE REVIEW

**Comment by Dreamy-Jazz** [comment]
That should work. notBlocked takes the opposite of whether someone is blocked. Therefore, I've changed the parameter name to isNotBlocked and swapped yes / no to reflect this.

**Review by GeneralNotability — CHANGES_REQUESTED** [review]

File: spihelper.js [hunk 1]
```diff
@@ -1070,7 +1071,7 @@ async function spiHelper_performActions() {
 | 1 = ${sockmasterName}
 | 2 = ${tag}
 | locked = ${isLocked}
-| notblocked = no
+| notblocked = ${isBlocked}
```
[the remark this answers is not in the document]
Comment (Dreamy-Jazz) [reply spihelper.js]: Done

File: spihelper.js [hunk 2]
```diff
@@ -1045,6 +1045,7 @@ async function spiHelper_performActions() {
 						break;
 				}
 				const isLocked = await spiHelper_isUserGloballyLocked(tagEntry.username) ? 'yes' : 'no';
+				const isNotBlocked = await spiHelper_getUserBlockReason(tagEntry.username) ? 'yes' : 'no';
```
[the remark this answers is not in the document]
Comment (Dreamy-Jazz) [reply spihelper.js:1048]: Fixed. The double negatives with the variable name confused me.

**Comment by Dreamy-Jazz** [comment]
I've made those two changes as noted by you.

**Review by GeneralNotability — APPROVED** [review]
Looks good to me! Thanks for the rework.

**Pull request merged**